portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -267,7 +267,7 @@ void vPortExitCritical( void )
 }
 /*-----------------------------------------------------------*/
 
-void vPortYieldFromISR( void )
+static void prvPortYieldFromISR( void )
```

Comment:
PLease add the declaration at the top of this file where other static functions are declared.